src/progress.ts

```diff
@@ -84,7 +84,7 @@ export class ProgressController {
       },
       log: (log: Log, message: string | Error) => {
         if (this._state === 'running') {
-          this._logRecording.push(message.toString());
+          this._logRecording.push(`[${log.name}] ${message.toString()}`);
```

Comment:
It is a bit weird that we are keeping two copies instead of using a custom formatter :/